tests/test_contexts.py

```diff
@@ -35,29 +33,29 @@ def block_checks_glb(name, glbs, body, obs=None):
     block = glbs[name]
     obs = obs or {}
     for k, v in obs.items():
-        yield assert_equal, v, glbs[k]
+        assert v == glbs[k]
     if isinstance(body, str):
         body = body.splitlines()
-    yield assert_equal, body, block.lines
-    yield assert_is, glbs, block.glbs
-    yield assert_is, None, block.locs
+    assert body == block.lines
+    assert glbs is block.glbs
+    assert block.locs is None
```

Comment:
> It is easy to get pytest to run all asserts and report a dot for each. We should just parameterize the tests, by moving the cases outside the functions. https://pytest.org/latest/parametrize.html

@melund I think that works for most cases.  However, for cases like this won't it create a whole new block object for each case?  The way this works on current master is that it checks consistency for a single block instance, rather than recreating many (hypothetically) identical block instances.